code/game/machinery/computer/vr_control.dm

```diff
@@ -0,0 +1,36 @@
+/obj/machinery/computer/vr_control
+	name = "VR console"
+	desc = "A list and control panel for all virtual servers."
+	icon_screen = "comm_logs"
+
+	light_color = LIGHT_COLOR_DARKGREEN
+
+
+/obj/machinery/computer/vr_control/New()
+	..()
+
+/obj/machinery/computer/vr_control/Destroy()
+	return ..()
+
+/obj/machinery/computer/vr_control/attack_ai(mob/user)
+	attack_hand(user)
+	ui_interact(user)
+
+
+/obj/machinery/computer/vr_control/attack_hand(mob/user)
+	add_fingerprint(user)
+	if(stat & (BROKEN|NOPOWER))
+		return
+	ui_interact(user)
+
+/obj/machinery/computer/vr_control/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1)
+	switch(alert("What would you like to do?", "VR Control", "Join Room", "Make Room", "Cancel"))
+		if("Join Room")
+			spawn_vr_avatar(user, input(user, "Choose a room to join.","Select Level") as null|anything in vr_rooms)
```

Comment:
Non-human mobs can use this (though not normally, since this is only in a closed-off space chunk), so you'll want to be careful